common/lib/xmodule/xmodule/services.py

```diff
@@ -119,3 +122,53 @@ def get_teams_configuration(self, course_id):
         if not self._course:
             self._course = self.get_course(course_id)
         return self._course.teams_configuration
+
+
+class SandboxService(Service):
```

Comment:
@symbolist FYI if you wanted this service to live in `xmodule.util.sandboxing` (and CacheService to live in `openedx.core.libcache_utils`), I can do that if I ensure that they are not subclasses of [`xblock.reference.plugins.Service`](https://github.com/edx/XBlock/blob/master/xblock/reference/plugins.py#L65). The [`pyfs` import](https://github.com/edx/XBlock/blob/master/xblock/reference/plugins.py#L21-L27) for that file causes an `django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.` exception when I try to move these declarations anywhere else.

Reply:
@symbolist Do you have a stance on whether these services should inherit from [`xblock.reference.plugins.Service`](https://github.com/edx/XBlock/blob/master/xblock/reference/plugins.py#L65), given the impact on where they can live in the code base?

Reply:
Umm, the superclass is not doing anything useful so either way is fine.

Reply:
Thanks @symbolist -- in that case, I'd rather have the services co-located with their related utility methods, so have moved them, see e3cbc5f and 3f61224.

Also merged latest master to see if this triggers the full test suite to run -- not sure why it isn't yet.

Reply:
👍🏽